### PR TITLE
Adds the Presentations page, navbar link, and metrics

### PR DESCRIPTION
**Pull request by jukent**
This PR does the infrastructure work for a separate presentations page.

Work to be done:
- replace presentation URLs with links to slide or poster pdfs (in VAST google drive or on GitHub?)

### CODE REVIEW

**Comment by jukent** [comment]
@clyne this should be good to go. It builds great for me and I've added my most recent talks.

**Comment by clyne** [comment]
This looks great, @jukent. Thanks! A couple of things: 

1. Can you update the contributor's guide to include presentations (currently only has info for presentations)
2. It doesn't seem like the metrics on the home page are being calculated correctly. It has 0 for presentations.

**Comment by jukent** [comment]
Okay I'll do that tomorrow.

I just set the metrics to 0 to create the field, but will be updated when the GH Actions runs (which it isn't for your local build).

**Comment by clyne** [comment]
got it. thanks!

**Comment by jukent** [comment]
Okay contributor's guide is updated!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -121,6 +121,24 @@ Edit `data/publications.yml` to add a new presentation or publication citation:
...
+      - Last, F.
+    conference: The Most Prestigious Conference
+    year: 2023
+    extra: City, ST
+    url: https://example.com
+````
+
+The `extra` field is optional and can contain conference city, virtual nature, or any other information.
+
+Please sepcify the `url` field to be a link to a PDF file of the talk slides or poster uploaded to the [VAST Presentations Google Drive fodler](https://drive.google.com/drive/folders/146ZZgV5Pb0-scadJtrPEXfP_jhDYwKVL). Please use a naming convention akin to `Last_Topic_ConferenceYYYY` and be sure that the file's sharing settings allow anyone to view the file.
```
Comment (clyne) [inline CONTRIBUTING.md]:
```suggestion
Please specify the `url` field to be a link to a PDF file of the talk slides or poster uploaded to the [VAST Presentations Google Drive fodler](https://drive.google.com/drive/folders/146ZZgV5Pb0-scadJtrPEXfP_jhDYwKVL). Please use a naming convention akin to `Last_Topic_ConferenceYYYY` and be sure that the file's sharing settings allow anyone to view the file.
```

**Review by clyne — COMMENTED** [review]
Looks great! Thanks for this, Julia. Just one typo to address.

**Review by clyne — APPROVED** [review]

**Pull request merged**